database/sql/src/main/scala/cromwell/database/slick/MetadataSlickDatabase.scala

```diff
@@ -208,7 +209,7 @@ class MetadataSlickDatabase(originalDatabaseConfig: Config)
           },
         summaryName = summarizeNameIncreasing
       )
-    } yield maximumMetadataEntryId
+    } yield (maximumMetadataEntryId - previousMaxMetadataEntryId, totalTableSize.toLong - maximumMetadataEntryId)
```

Comment:
The second calculation is comparing apples and oranges.

- The `totalTableSize` is the number of rows in the table.
- The `maximumMetadataEntryId` is the max primary key id.

Check out these values in alpha.

```shell
$ <toolbox-connect> -e alpha "
SELECT TABLE_ROWS FROM INFORMATION_SCHEMA.TABLES WHERE TABLE_NAME = 'METADATA_ENTRY';
SELECT SUMMARY_POSITION FROM SUMMARY_STATUS_ENTRY WHERE SUMMARY_NAME = 'WORKFLOW_METADATA_SUMMARY_ENTRY_INCREASING';
SELECT MAX(METADATA_JOURNAL_ID) FROM METADATA_ENTRY;
"
TABLE_ROWS
2818962057
SUMMARY_POSITION
3543925173
MAX(METADATA_JOURNAL_ID)
3543925179
```

Btw, the `select count(1)` generated by slick is going to be _slow_, probably timing out.
- https://dba.stackexchange.com/questions/130780/mysql-count-performance/130851
- http://slick.lightbend.com/doc/3.2.3/queries.html#aggregation
- https://stackoverflow.com/questions/5179969/what-is-better-in-mysql-count-or-count1#answer-5180023

TL;DR: You probably want to replace checking the table size with a query to get the max id in `METADATA_ENTRY`.